sdk/python/feast/cli.py

```diff
@@ -246,173 +170,46 @@ def _get_labels_dict(label_str: str) -> Dict[str, str]:
-    type=click.Path(exists=True),
-)
-def feature_table_create(filename):
-    """
-    Create or update a feature table
-    """
-
-    feature_tables = [
-        FeatureTable.from_dict(ft_dict) for ft_dict in yaml_loader(filename)
-    ]
-    feast_client = Client()  # type: Client
-    feast_client.apply(feature_tables)
-
-
-@feature_table.command("describe")
-@click.argument("name", type=click.STRING)
-@click.option(
-    "--project",
-    "-p",
-    help="Project that feature table belongs to",
-    type=click.STRING,
-    default="default",
-)
-def feature_table_describe(name: str, project: str):
-    """
-    Describe a feature table
-    """
-    feast_client = Client()  # type: Client
-    ft = feast_client.get_feature_table(name=name, project=project)
-
-    if not ft:
-        print(f'Feature table with name "{name}" could not be found')
-        return
-
-    print(yaml.dump(yaml.safe_load(str(ft)), default_flow_style=False, sort_keys=False))
-
-
-@feature_table.command(name="list")
-@click.option(
-    "--project",
-    "-p",
-    help="Project that feature table belongs to",
-    type=click.STRING,
-    default="",
-)
-@click.option(
-    "--labels",
-    "-l",
-    help="Labels to filter for feature tables",
-    type=click.STRING,
-    default="",
-)
-def feature_table_list(project: str, labels: str):
-    """
-    List all feature tables
-    """
-    feast_client = Client()  # type: Client
-
-    labels_dict = _get_labels_dict(labels)
-
-    table = []
-    for ft in feast_client.list_feature_tables(project=project, labels=labels_dict):
-        table.append([ft.name, ft.entities])
-
-    from tabulate import tabulate
-
-    print(tabulate(table, headers=["NAME", "ENTITIES"], tablefmt="plain"))
-
-
-@cli.group(name="projects")
-def project():
-    """
-    Create and manage projects
-    """
-    pass
-
-
-@project.command(name="create")
-@click.argument("name", type=click.STRING)
-def project_create(name: str):
-    """
-    Create a project
-    """
-    feast_client = Client()  # type: Client
-    feast_client.create_project(name)
-
-
-@project.command(name="archive")
-@click.argument("name", type=click.STRING)
-def project_archive(name: str):
-    """
-    Archive a project
-    """
-    feast_client = Client()  # type: Client
-    feast_client.archive_project(name)
-
-
-@project.command(name="list")
-def project_list():
-    """
-    List all projects
-    """
-    feast_client = Client()  # type: Client
-
-    table = []
-    for project in feast_client.list_projects():
-        table.append([project])
-
-    from tabulate import tabulate
-
-    print(tabulate(table, headers=["NAME"], tablefmt="plain"))
-
-
 @cli.command("apply")
-@click.argument(
-    "repo_path", type=click.Path(dir_okay=True, exists=True), default=Path.cwd
-)
-def apply_total_command(repo_path: str):
+def apply_total_command():
     """
-    Applies a feature repo
+    Apply a feature repo
```

Comment:
Should we have a more descriptive message here? `Create or update a feature store deployment`?

Reply:
We have couple of commands that use `X a feature repo` as a help message:
```
  apply                    Apply a feature repo
  init                     Initialize a feature repo
  teardown                 Tear down infra for a feature repo
```

For consistency, how about changing them to
```
  apply                    Create or update a feature store deployment
  init                     Initialize a feature store deployment
  teardown                 Tear down deployed feature store infrastructure 
```
?

Reply:
Sure